src/commands/generateProject.ts

```diff
@@ -60,9 +60,12 @@ export async function generateProject(): Promise<void> {
       return;
     }
 
-    const targetFolder = await prompts.askForFolder({
-      openLabel: "Generate into this folder",
-    });
+    const targetFolder = await prompts.askForTargetFolder(
+      {
+        openLabel: "Generate into this folder",
```

Comment:
Now that we have a separate askForTargetFolder method instead of just the generic askForFolder, I think we can hide the OpenDialogOptions and move them into askForTargetFolder.